.gitignore

```diff
@@ -3,3 +3,4 @@
 
 # Autogenerated Blender files
 *.blend1
+*.ini
```

Comment:
I'll give it a full review tomorrow but what is an .ini? I've never seen that file type before, at least not from Blender.

Reply:
its a windows data file. Honestly I dont know why its in the gitignore but its not like our art repo needs .inis anyway

Reply:
You didn't add it to the .gitignore? Did it happen automatically?

Sure the repo doesn't need it, but it also doesn't need many other file types.

Unless one of the programs you are using is actually generating .inis and saving them in your local repo directory, then there is no reason to add it to the .gitignore.